src/connection.ts

```diff
@@ -936,6 +939,16 @@ class Connection extends EventEmitter {
         this.config.options.useUTC = config.options.useUTC;
       }
 
+      if (config.options.validateBulkLoadParameters !== undefined) {
+        if (typeof config.options.validateBulkLoadParameters !== 'boolean') {
+          throw new TypeError('The "config.options.validateBulkLoadParameters" property must be of type boolean.');
+        }
+
+        this.config.options.validateBulkLoadParameters = config.options.validateBulkLoadParameters;
+      } else {
+        deprecate('The default value for "config.options.validateBulkLoadParameters" will change from `false` to `true` in the next major version of `tedious`. Set the value to `true` or `false` explicitly to silence this message.');
```

Comment:
Will this be a better message ? "validateBulkLoadParameters" property will be depercated in the next major version of `tedious`."

Reply:
I think we'll want this to be a 2 step process:

* First, we introduce the `validateBulkLoadParameters` flag, but keep defaulting to `false`. (minor release)
* Then, we switch the default to `true`, and deprecate setting the `validateBulkLoadParameters` flag. (major release)
* Eventually, we drop the `validateBulkLoadParameters` flag. (major release)